docs: Remove legacy EPID instructions (#916)
oasisprotocol/docs: docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

can facilitate various SGX services such as launch approval, remote attestation
quote signing, etc.

> Oasis node supports the (newer) DCAP attestation method.
> 
> To see if your system supports DCAP attestation run the following:

Reviewer: ```suggestion
Oasis node requires the use of DCAP attestation. To see if your system supports it, run the following:
```

> 

Reviewer: Drop the above tip for legacy SGX drivers.

:::tip

> For DCAP use the `ghcr.io/oasisprotocol/aesmd-dcap:master` docker image.

Reviewer: This tip is no longer needed as the example already shows the use of this image. Just remove it.
